### PR TITLE
Docs: Update _index.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Added a link

### CODE REVIEW

File: docs/sources/features/datasources/_index.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ In addition to the data sources that you have configured in your Grafana, there
 - **Mixed -** Select this to query multiple data sources in the same panel. When this data source is selected, Grafana allows you to select a data source for every new query that you add.
   * The first query will use the data source that was selected before you selected **Mixed**.
   * You cannot change an existing query to use the Mixed Data Source.
+  * Graphana Play example: [Mixed data sources](https://play.grafana.org/d/000000100/mixed-datasources?orgId=1)
```
Comment (aknuds1) [inline docs/sources/features/datasources/_index.md:44]: Assuming Graphana is a typo:
```suggestion
  * Grafana Play example: [Mixed data sources](https://play.grafana.org/d/000000100/mixed-datasources?orgId=1)
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
Spotted a typo.

**Review by aknuds1 — CHANGES_REQUESTED** [review]
GitHub leaves duplicated reviews for me sometimes.

File: docs/sources/features/datasources/_index.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/features/datasources/_index.md:44]: Some days...

**Review by aknuds1 — APPROVED** [review]

**Pull request merged**